yaml/thz504.yaml

```diff
@@ -5,3 +5,6 @@ esphome:
 
 packages:
   BASE: !include { file: wp_base.yaml}
+  
+  ABLUFTTAUPUNKT:                 !include { file: thz504_temperature.yaml, vars: { property: "ABLUFTTAUPUNKT" }} 
```

Comment:
You need to define the newly added values in property.h, otherwise this will not compile